Documentation/ApiOverview/SiteHandling/SiteSettingDefinitions.rst

```diff
@@ -8,88 +8,187 @@ Site settings definitions
+default value. They can be defined in :ref:`site-sets`, in a file called
+:file:`_blog_settins.definitions.yaml`.
 
-TYPO3 wants to remedy this with settings definitions and now provides an API
-with which you can add additional descriptive definitions for each individual
-site setting.
+It is recommended to use site-sets and their UI configuration in favor of
+TypoScript Constants.
 
-..  literalinclude:: _Settings/_settings.definitions.yaml
-    :caption: EXT:my_extension/Configuration/Sets/MySet/settings.definitions.yaml
+All available settings are displayed in the :ref:`site-settings-editor`.
+
+The site settings provided by an extension can be automatically documented in the
+extensions manual, see
+:ref:`site settings documentation <h2document:reference-site-settings>`.
+
+..  contents:: Table of contents
+
+..  _site-settings-definition-example:
+
+Site setting definition example
+===============================
+
+..  literalinclude:: _Settings/_blog_settings.definitions.yaml
+    :caption: EXT:blog_example/Configuration/Sets/BlogExample/settings.definitions.yaml (Excerpt)
+
+See the complete example at
+`settings.definitions.yaml (GitHub) <https://github.com/TYPO3-Documentation/blog_example/blob/main/Configuration/Sets/BlogExample/settings.definitions.yaml>`__.
+
+..  figure:: /Images/ManualScreenshots/SiteHandling/SiteSettingsDefinition.png
+    :alt: Screenshot demonstration the position of the categories, labels etc
+
+    The parts marked by a number can be configured, see list bellow
+
+
+..  _site-settings-definition-properties:
+
+Site setting definition properties
+==================================
+
+..  confval-menu::
+    :display: table
+    :name: site-setting-definition
+    :type:
+    :required:
+
+    ..  confval:: categories
+        :type: array
+        :name: site-settings-definition-categories
+
+        ..  confval:: label
+            :type: string
+            :name: site-settings-definition-categories-label
+
+        ..  confval:: parent
+            :type: :confval:`site-settings-definition-categories` key
+            :name: site-settings-definition-categories-parent
+
+    ..  confval:: settings
+        :type: array
+        :name: site-settings-definition-settings
+
+        ..  confval:: label
+            :type: string
+            :name: site-settings-definition-settings-label
+
+        ..  confval:: description
+            :type: string
+            :name: site-settings-definition-settings-description
+
+        ..  confval:: category
+            :type: :confval:`site-settings-definition-categories` key
+            :name: site-settings-definition-settings-category
+
+        ..  confval:: type
+            :type: a :ref:`definition type <definition-types>`
+            :name: site-settings-definition-settings-type
+            :required:
+
+        ..  confval:: default
+            :type: mixed
+            :name: site-settings-definition-settings-default
+            :required:
```

Comment:
You add a column "required" and marked "default" as "required", but this info was not rendered in column above

Reply:
it is indeed required to have a default value.